storagedriver/factory/factory.go

```diff
@@ -58,7 +58,12 @@ func Create(name string, parameters map[string]interface{}) (storagedriver.Stora
 		// }
 		// return driverClient, nil
 	}
-	return driverFactory.Create(parameters)
+	d, err := driverFactory.Create(parameters)
```

Comment:
The `New` method should produce a test-compliant instance. The usage of the wrapper is enforced through passing unit tests.

A small amount of duplication is not the end of the world.